hsds/link_dn.py

```diff
@@ -109,6 +114,16 @@ async def GET_Links(request):
         titles.sort()  # sort by key
```

Comment:
`globmatch` returns the link titles in arbitrary order, so it should probably be moved before this